README.md

```diff
@@ -2,50 +2,46 @@
-
-### July 5
-
-On July 5, we hosted a [datathon workshop](http://www.meetup.com/DataPhilly/events/232137908/"DataPhilly & Code for Python July 5 Datathon Meetup") introducing Neo4j and this project. [**@gwaygenomics**](https://github.com/gwaygenomics "Greg Way on GitHub") presented the biological motivation behind Project Cognoma ([slides](https://www.dropbox.com/s/4btewka0icgse4f/DatathonWorkshop_5July2016_Final.pptx?dl=0)) and [**@dhimmel**](https://github.com/dhimmel "Daniel Himmelstein on GitHub") discussed Hetionet and the Cypher query language ([slides](http://slides.com/dhimmel/datathon)).
-
-Thanks everyone who attended. The response, excitement, and diverse talents were highly encouraging. This following week, we'll work to hammer out project details including:
-
-+ information on contributing and getting involved
-+ a more detailed project outline
-+ specifications for the various project components
-+ a list of skillsets needed
-+ a higher-level description of project goals
-
-Check back here for more details. In the meantime, feel free to open issues with feedback, suggestions, or questions. This is a community oriented project, so don't hesitate to chime in — you can [introduce yourself here](https://github.com/cognoma/cognoma/issues/2 "Issue #2: Introduce yourself here to get involved").
-
-## Implementation Details
-
-We're beginning to flesh out implementation details. Discuss the software architecture in [Issue 5](https://github.com/cognoma/cognoma/issues/5 "Issue #5: Designing the software architecture for Project Cognoma") and technical ideas in [Issue 4](https://github.com/cognoma/cognoma/issues/4 "Issue #4: Technical Ideas"). The following diagram is the current draft of the project's software architecture:
-
-![Diagram of the Cognoma software architecture](architecture.png?raw=true)
++ **Cancer Data**: Claire McLeod ([@**clairemcleod**](https://github.com/clairemcleod))
++ **Machine Learning**: Yichuan Liu ([@**yl565**](https://github.com/yl565))
++ **Backend**: Andrew Madonna ([@**awm33**](https://github.com/awm33)), Derek Goss ([@**dcgoss**](https://github.com/dcgoss)), Alan Elkner ([@**aelkner**](https://github.com/aelkner))
++ **Frontend**: Bonnie DiPasquale ([@**bvasko**](https://github.com/bvasko)), Benjamin Dolly ([@**bdolly**](https://github.com/bdolly))
++ **Community**: Karin Wolok ([@**KarinSpiderwoman**](https://github.com/KarinSpiderwoman))
++ **Wildcards**: Gregory Way ([@**gwaygenomics**](https://github.com/gwaygenomics)), Casey Greene ([@**cgreene**](https://github.com/cgreene)), Daniel Himmelstein ([@**dhimmel**](https://github.com/dhimmel))
```

Comment:
yes! love the title. Makes me think of [Charlie Kelly](https://www.youtube.com/watch?v=MYtjpIwamos) (possibly NSFW (language) ?)